tests/pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 ; capture=no will uglify the output, but the stdin epoll test fails without.
```

Comment:
The comment is no longer necessary as `--capture=no` was removed.

Reply:
Done.